app/views/shared/vote/_delete.html.erb

```diff
@@ -0,0 +1,3 @@
+<%= link_to vote, method: :delete, id: "delete" do %>
```

Comment:
what does the `id: "delete"` accomplish here?

Reply:
ah, it's probably setting the HTML attribute? carry on if that's the case.

Reply:
exactly